src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -25,122 +26,31 @@ public DeploymentOptionProperty(T value, string description = "", string display
-
-        public DeploymentOptionProperty<bool> IgnoreFullTextCatalogFilePath { get; set; }
-
-        public DeploymentOptionProperty<bool> IgnoreWhitespace { get; set; }
-
-        public DeploymentOptionProperty<bool> IgnoreWithNocheckOnForeignKeys { get; set; }
-
-        public DeploymentOptionProperty<bool> VerifyCollationCompatibility { get; set; }
-
-        public DeploymentOptionProperty<bool> UnmodifiableObjectWarnings { get; set; }
-
-        public DeploymentOptionProperty<bool> TreatVerificationErrorsAsWarnings { get; set; }
-
-        public DeploymentOptionProperty<bool> ScriptRefreshModule { get; set; }
-
-        public DeploymentOptionProperty<bool> ScriptNewConstraintValidation { get; set; }
-
-        public DeploymentOptionProperty<bool> ScriptFileSize { get; set; }
-
-        public DeploymentOptionProperty<bool> ScriptDeployStateChecks { get; set; }
-
-        public DeploymentOptionProperty<bool> ScriptDatabaseOptions { get; set; }
-
-        public DeploymentOptionProperty<bool> ScriptDatabaseCompatibility { get; set; }
-
-        public DeploymentOptionProperty<bool> ScriptDatabaseCollation { get; set; }
-
-        public DeploymentOptionProperty<bool> RunDeploymentPlanExecutors { get; set; }
-
-        public DeploymentOptionProperty<bool> RegisterDataTierApplication { get; set; }
-
-        public DeploymentOptionProperty<bool> PopulateFilesOnFileGroups { get; set; }
-
-        public DeploymentOptionProperty<bool> NoAlterStatementsToChangeClrTypes { get; set; }
-
-        public DeploymentOptionProperty<bool> IncludeTransactionalScripts { get; set; }
-
-        public DeploymentOptionProperty<bool> IncludeCompositeObjects { get; set; }
-
-        public DeploymentOptionProperty<bool> AllowUnsafeRowLevelSecurityDataMovement { get; set; }
-
-        public DeploymentOptionProperty<bool> IgnoreWithNocheckOnCheckConstraints { get; set; }
-
-        public DeploymentOptionProperty<bool> IgnoreFillFactor { get; set; }
-
-        public DeploymentOptionProperty<bool> IgnoreFileSize { get; set; }
-
-        public DeploymentOptionProperty<bool> IgnoreFilegroupPlacement { get; set; }
-
-        public DeploymentOptionProperty<bool> DoNotAlterReplicatedObjects { get; set; }
-
-        public DeploymentOptionProperty<bool> DoNotAlterChangeDataCaptureObjects { get; set; }
-
-        public DeploymentOptionProperty<bool> DisableAndReenableDdlTriggers { get; set; }
-
-        public DeploymentOptionProperty<bool> DeployDatabaseInSingleUserMode { get; set; }
-
-        public DeploymentOptionProperty<bool> CreateNewDatabase { get; set; }
-
-        public DeploymentOptionProperty<bool> CompareUsingTargetCollation { get; set; }
-
-        public DeploymentOptionProperty<bool> CommentOutSetVarDeclarations { get; set; }
-
         // Command timeout to 120 seconds when executing queries against SQL Server.
         public DeploymentOptionProperty<int> CommandTimeout { get; set; } = new DeploymentOptionProperty<int>(120);
```

Comment:
these timeouts and additional deployment contributor options aren't in mssql.d.ts in ADS, probably since they aren't being used there. It would be helpful for there to be a comment in this file why they are here and not in ADS. Or if they aren't needed, remove them for now.